services/core-api/app/templates/email/projects/emli_project_summary_email.html

```diff
@@ -1,114 +1,144 @@
-            </div>
-            <!-- START CENTERED WHITE CONTAINER -->
-            <table role="presentation" class="main" style=" border-width:1px; border-style:solid; border-color:#DED9D9; border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background: #ffffff; border-radius: 3px; width: 100%;" width="100%">
-              
-              <!-- START MAIN CONTENT AREA -->
+      table.body .btn a {
+        width: 100% !important;
+      }
+
+      table.body .img-responsive {
+        height: auto !important;
+        max-width: 100% !important;
+        width: auto !important;
+      }
+    }
+  </style>
+</head>
+
+<body
+  style="background-color: #ffffff; font-family: sans-serif; -webkit-font-smoothing: antialiased; font-size: 14px; line-height: 1.4; margin: 0; padding: 0; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;">
+  <table role="presentation" cellpadding="0" cellspacing="0" class="body"
+    style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #ffffff; width: 100%;"
+    width="100%">
+    <tr>
+      <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;" valign="top">&nbsp;</td>
+      <td class="container"
+        style="font-family: sans-serif; font-size: 14px; vertical-align: top; display: block; max-width: 580px; padding: 10px; width: 580px; margin: 0 auto;"
+        width="580" valign="top">
+        <div class="content"
+          style="box-sizing: border-box; display: block; margin: 0 auto; max-width: 580px; padding: 10px;">
+          <div class="content-block"
+            style="font-family: sans-serif; vertical-align: top; padding-bottom: 36px; padding-top: 10px; font-size: 12px; text-align: center;"
+            valign="top">
+            <img src="{{core_logo}}" width="25%" style="object-fit: contain;">
+          </div>
+          <!-- START CENTERED WHITE CONTAINER -->
+          <table role="presentation" class="main"
+            style=" border-width:1px; border-style:solid; border-color:#DED9D9; border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background: #ffffff; border-radius: 3px; width: 100%;"
+            width="100%">
+
+            <!-- START MAIN CONTENT AREA -->
+            <tr>
+              <td class="wrapper" style="vertical-align: top; box-sizing: border-box; padding: 20px;" valign="top">
+                <table role="presentation" cellpadding="0" cellspacing="0"
+                  style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;"
+                  width="100%">
+                  <tr>
+                    <td
+                      style="vertical-align: top; text-align: center; font-family: sans-serif; font-size: 24px; font-weight: 700; line-height: 30px; color: #5E46A1; padding: 16px 0px 24px;"
+                      valign="top">
+                      {{message}}
```

Comment:
This is the only change (making message dynamic). Rest is formatting.

Reply:
I really really want to create some reusable email templates with something like handlebars...

One day... 🙏